### PR TITLE
CHAAAAAAAAAAAOOOOOOOOOSSSSSSS

### PR DESCRIPTION
**Pull request by EtherTyper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,5 @@
 # ChaosBot
 
-> Chaos, the vacant and infinite space which existed according to the ancient
-> cosmogonies previous to the creation of the world, and out of which the gods,
-> men, and all things arose.
-
-ChaosBot is a social coding experiment to see what happens when the absolute
-direction of a software project is turned over to the open source community.
+### _CHAAAAAAAAAAAOOOOOOOOOSSSSSSS_
```
Comment (sugar700) [inline README.md:3]: Shouldn't be italics and all-caps

File: README.md [hunk 2]
```diff
@@ -1,83 +1,5 @@
...
 
 ![Image of a kitten with overlay: "I WONDER IF BEING SANE MEANS DISREGARDING THE CHAOS THAT IS LIFE...PRETENDING ONLY AN INFINITESIMAL SEGMENT OF IT IS REALITY" Source: http://existentialpets.tumblr.com/post/48069725344](assets/chaos-kitty.jpg)
```
Comment (sugar700) [inline README.md:5]: Left kitten alone

File: README.md [hunk 3]
```diff
@@ -1,83 +1,5 @@
...
-
-## How it works
```
Comment (sugar700) [inline README.md:12]: How are people going to know how to use the bot if there are no instructions.

File: README.md (hunk 1 above)
Comment (sugar700) [reply README.md:3]: Or you know, should be italics and all caps twice. Like, twice as upper case characters.

File: README.md [hunk 4]
```diff
@@ -1,83 +1,5 @@
...
-  depending on the local server time.
-* While the voting process is going, users can change their reactions and edit
-  their comments, effectively changing their vote.
-
-## Death Counter
-
-Chaosbot has died 4 times.  This counter is incremented whenever the trunk breaks
-and the server must be restarted manually.
-
-## Server details
```
Comment (sugar700) [inline README.md:58]: How are users going to know server parameters

**Review by sugar700 — APPROVED** [review]
Otherwise fine, upvoting just because this will be rejected anyway.

**Comment by rvegas** [comment]
👎

**Comment by eamanu** [comment]
(N)

**Comment by Shuunen** [comment]
👎

**Review by jblz — CHANGES_REQUESTED** [review]
mean <> chaotic :-1:

**Comment by ozyx** [comment]
Why hasn't this been closed yet? :|

**Comment by reddraggone9** [comment]
@ozyx 

This can't be merged or closed by votes because there are still conflicts which need to be resolved. It will eventually be closed for being stale. Right now, [that code][0] looks at the [last push to the originating repo][1], which was [10 hours ago][2]. With the current code, it needs [24 hours][3] before it will be closed as stale.

#271 will instead have Chaos look at the last push to *this particular PR* instead.

[0]: https://github.com/chaosbot/Chaos/blob/4a88895b058e9ada04be023443d7d458870f64e9/github_api/prs.py#L150-L155
[1]: https://github.com/chaosbot/Chaos/blob/4a88895b058e9ada04be023443d7d458870f64e9/github_api/prs.py#L104-L112
[2]: https://github.com/EtherTyper/chaos/commit/e44bdb5a14305008e2111e445d8b0a8bd4c80f9c
[3]: https://github.com/chaosbot/Chaos/blob/4a88895b058e9ada04be023443d7d458870f64e9/settings.py#L69-L71

**Pull request closed without merging**